public_suffix_list.dat

```diff
@@ -11204,6 +11172,80 @@ us-2.evennode.com
 // Submitted by Peter Ruibal <public-suffix@fb.com>
 apps.fbsbx.com
 
+// FAITID : https://faitid.org/ru/node/346
```

Comment:
Hello guys,
Any progress on this pull request? It's very disappointing that RU community can't use features provided by PublicList.org till now :(